src/packages/cli/package.json

```diff
@@ -62,6 +62,8 @@
     "typescript": "4.6.4"
   },
   "dependencies": {
+    "table": "6.8.0",
+    "ps-list": "7.2.0",
```

Comment:
I've dropped out that dependency as you suggest. We'll need to readdress this if we decide that it's important to ensure that the process with that id is the once we are looking for.
![image](https://user-images.githubusercontent.com/6363749/191770426-439c0b0f-7516-49e7-b047-3c2c77a01449.png)

Reply:
As for the concerns about executing an .exe - I'm not really sure. Is it wildly different to executing any other binaries (such as the websockets??)

Maybe @davidmurdoch has an opinion? Otherwise I can research.

Reply:
Why are you using an old version?

Reply:
I'm using an old version as the next version removes commonjs support. I've created a fork that:
- Supports CJS
- Uses the latest non-Windows implementation
- Uses the native `task-list` Windows utility (rather than `fast-list`)

Which can be found here: https://github.com/trufflesuite/ps-list

Once the pulls on that are merged, I'll publish, and add the dependency here.